src/database/model/realtimeDataModel.ts

```diff
@@ -0,0 +1,25 @@
+import {Model} from '@nozbe/watermelondb';
+import {field, date} from '@nozbe/watermelondb/decorators';
+
+export default class RealtimeDataModel extends Model {
+  static table = 'realtime_data';
+
+  @field('distance') distance!: number;
+  @field('elapsed_time') elapsed_time!: number;
+  @field('speed') speed!: number;
+  @field('latitude') latitude!: number;
+  @field('longitude') longitude!: number;
+  @field('altitude') altitude!: number;
+  @field('heading') heading!: number;
+  @field('heart_rate') heart_rate!: number;
+  @field('instant_power') instant_power!: number;
+  @field('three_sec_power') three_sec_power!: number;
+  @field('ten_sec_power') ten_sec_power!: number;
+  @field('cadence') cadence!: number;
+  @field('is_bluetooth') is_bluetooth!: boolean;
+  @field('is_wifi') is_wifi!: boolean;
```

Comment:
What is the `!` for isn't that just the field name or does it have some meaning within watermelon db?

Reply:
Also their docs suggest using snake_case for db names and camel_case for javascript objects. Should we just go with that so we don't have to think about it? 

> Note: Why do I have to type the field/column name twice? The database convention is to use snake_case for names, and the JavaScript convention is to use camelCase. So for any multi-word name, the two differ. Also, for resiliency, we believe it's better to be explicit, because over time, you might want to refactor how you name your JavaScript field names, but column names must stay the same for backward compatibility.

Reply:
Not 100% sure about the `!` . I had copied it from an example, I assume it means required column or something. I can remove it for now and see how things go. 

Also agree on using camelCase for the js objects, had not read that in their docs 👍

Reply:
oh interesting, when i remove the `!` i get `Property 'distance' has no initializer and is not definitely assigned in the constructor.`  so its actually doing something else